File: BMEN3310_Project.m
Valve = csvread('ValveSensor.csv');
ECG = csvread('ECG.csv');

figure('Name','ECG compared to Hall Sensor Signal','NumberTitle','off');

subplot(2,1,1);
plot(ECG);
title("ECG"); ylabel("Amplitude (mV)"); xlabel("Time");
xlim([ 0 5000]);

subplot(2,1,2);
plot(Valve);
title("Signal from Valve Sensor"); ylabel("Amplitude (mV)"); xlabel("Time");
xlim([ 0 5000]);

File: BMEN3310_Project.py
import matplotlib
import numpy as np
import pandas as pd
from matplotlib import pyplot as plt


def main():
    valve = pd.read_csv('ValveSensor.csv'); 
    ECG = pd.read_csv('ECG.csv');           
        
    #plotting the data to the figure
    plt.subplot(2, 1, 1);
    plt.plot(ECG);
    plt.title("ECG"); plt.ylabel("Amplitude (mV)"); plt.xlabel("Time");
    plt.xlim(0, 5000);
    
    plt.subplot(2, 1, 2);
    plt.plot(valve);
    plt.title("Signal from Valve Sensor"); plt.ylabel("Amplitude (mV)"); plt.xlabel("Time");
    plt.xlim(0, 5000);
    plt.show(); 

if __name__ == "__main__":
    main()
    